docs/site/Extending-OpenAPI-specification.md

```diff
@@ -0,0 +1,154 @@
+---
+lang: en
+title: 'Extending OpenAPI Specification'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Extending-OpenAPI-specification.html
+---
+
+## OpenAPI Specification Enhancer
+
+The APIs in a LoopBack `RestApplication` are described by the
+[OpenAPI Specification(short for OAS)](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md).
+An application's OAS are mainly generated from
+[controllers](https://loopback.io/doc/en/lb4/Controllers.html) and their
+members' metadata, but we would also like to contribute specifications from
+other places. Therefore an extension point `OASEnhancerService` is created to
+allow registered extensions to provide their OAS fragments and modify a rest
+application's specification.
+
+_Read about the extension point/extension patten in
+[documentation](https://loopback.io/doc/en/lb4/Extension-point-and-extensions.html)_
```

Comment:
suggestion: https://loopback.io/doc/en/lb4/Extension-point-and-extensions.html -> `Extension-point-and-extensions.md`

Reply:
good catch